contrib/build-push-ceph-container-imgs.sh

```diff
@@ -76,10 +76,13 @@ function install_docker {
     sudo apt-get update
     sudo env DEBIAN_FRONTEND=noninteractive apt-get install -y --force-yes docker-ce
   fi
-  # ignore failure here; some distros (el8) don't need this
-  sudo systemctl start docker || true
-  sudo systemctl status --no-pager docker || true
-  sudo chgrp "$(whoami)" /var/run/docker.sock || true
+  sudo systemctl start docker
+  sudo systemctl status --no-pager docker
+  sudo chgrp "$(whoami)" /var/run/docker.sock
+}
+
+function install_podman {
+  sudo dnf install -y podman
```

Comment:
I think I need podman-docker package too right ?

Reply:
For `docker` commands to work, yes.